server/routes/comments.js

```diff
@@ -1,12 +1,12 @@
 /* eslint-disable eol-last */
 /* eslint-disable import/first */
-// import express from 'express';
+import express from 'express';
 
-// const router = express.Router();
+const router = express.Router();
 
-// import Admin from '../controller/admin';
+import Comments from '../controller/comments';
 
-// router.post('/auth/signup', Admin.createnewAdmin);
+router.post('/auth/signup', Comments.createComment);
```

Comment:
Do you think it is appropriate to map `signup` route to `createComment`?

Reply:
I have actually cleaned this up at the moment of posting this comment